mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
```

Comment:
I don't think it's necessary in this case, the main code is actually just a call to an already existing mne function, no need to create a mixin class for this, especially if one of the object is going to be deprecated